src/sonic-config-engine/minigraph.py

```diff
@@ -577,11 +577,11 @@ def parse_dpg(dpg, hname):
                 vlan_attributes['alias'] = vintfname
             vlans[sonic_vlan_name] = vlan_attributes
 
-        dhcp = child.find(str(QName(ns, "Dhcp")))
-        dhcp_table = {}
+        dhcp_relay = child.find(str(QName(ns, "DhcpRelay")))
```

Comment:
Sorry, I missed that earlier. There will no new node in the minigraph with DhcpRelay name. The server/helper IPs will be provided as follows:

```
<VlanInterfaces>
<VlanInterface>
<ElementType>VlanInterface</ElementType>
<Name>Vlan2</Name>
<AttachTo>Ethernet2/1;Ethernet2/3;Ethernet4/1;Ethernet4/3;Ethernet6/1;Ethernet6/3;Ethernet12/1;Ethernet12/3;Ethernet14/1;Ethernet14/3;Ethernet16/1;Ethernet16/3;Ethernet18/1;Ethernet18/3;Ethernet20/1;Ethernet20/3;Ethernet22/1;Ethernet22/3;Ethernet28/1;Ethernet28/3;Ethernet30/1;Ethernet30/3;Ethernet32/1;Ethernet32/3;Ethernet1/1;Ethernet3/1;Ethernet5/1;Ethernet11/1;Ethernet13/1;Ethernet15/1;Ethernet17/1;Ethernet19/1;Ethernet21/1;Ethernet27/1;Ethernet29/1;Ethernet31/1</AttachTo>
<FhrpProtoType i:nil="true"/>
<Type i:nil="true"/>
<DhcpRelays>13.107.194.100;13.107.194.101;13.107.194.102;13.107.194.103;10.254.32.207;10.254.32.223;10.254.32.95;10.254.32.127;10.254.32.191;10.254.32.159;10.254.45.31;10.254.45.63</DhcpRelays>
<Dhcpv6Relays>2603:10e1:0:1::;2603:10e1:0:1::1;2603:10e1:0:1::2;2603:10e1:0:1::3</Dhcpv6Relays>
<VlanID>2</VlanID>
<Tag>2</Tag>
<Subnets>25.99.157.0/26</Subnets>
<MacAddress i:nil="true"/>
<SecondarySubnets/>
</VlanInterface>
</VlanInterfaces>
```